include/nuttx/compiler.h

```diff
@@ -472,6 +472,13 @@
 #    define no_builtin(n)
 #  endif
 
+/* CMSE extention */
+
+#  ifdef CONFIG_ARCH_HAVE_TRUSTZONE
+#    define cmse_nonsecure_entry __attribute__((cmse_nonsecure_entry))
+#    define cmse_nonsecure_call __attribute__((cmse_nonsecure_call))
```

Comment:
Do we need to add similar empty defines for other compilers?
Is this going to be used in common code?

Reply:
I think this only has use in armv8-m specific code.